EKSD_LATEST_RELEASES

```diff
@@ -33,12 +33,16 @@ releases:
 - branch: 1-28
   kubeVersion: v1.28.15
   number: 38
+  endOfStandardSupport: 12/31/2024
```

Comment:
Maybe we should use the unambiguous format of `yyyy-mm-dd` here to avoid confusion?

Reply:
Agreed, though it needs to be double-quoted in order to parse it as a string when unmarshalling it.
Ref:- https://discourse.getgrav.org/t/international-yaml-date-field/3479/4